.circleci/config.yml

```diff
@@ -1,34 +0,0 @@
-version: 2.1
-
-orbs:
-  # Using 1.0 of our Orbs means it will use the latest 1.0.x version from https://github.com/wordpress-mobile/circleci-orbs
-  ios: wordpress-mobile/ios@1.0
-
-# YAML anchors for some common/repeated values
-x-common-params:
-  - &xcode-version "12.5.1"
-  - &podspec "WordPressKit.podspec"
-  - &on-tags-only
-      tags:
-        only: /.*/
-      branches:
-        ignore: /.*/
-
-workflows:
-  test_and_validate:
-    jobs:
-      - ios/test:
-          name: Test
-          xcode-version: *xcode-version
-          workspace: WordPressKit.xcworkspace
-          scheme: WordPressKitTests
-          device: iPhone 12
-          ios-version: "14.5"
-          bundle-install: true
-          pod-install: true
```

Comment:
~This job doesn't seem to have been ported to BuildKite's pipeline?~

nvm it was ported before this PR (thus hidden in this PR's diff, which is why I missed it) and is properly present on line 16 on the pipeline